index.js

```diff
@@ -26,6 +26,12 @@ var createSwaggerPage = (options) => {
         },
         apis: options.apis || []
     });
+	//Add any external definitions provided
```

Comment:
You should verify that options.definitions exist before your `for` loop (because it's an optional parameter), with something like this :
```js
if (options.definitions) {
```